academic/editFM.py

```diff
@@ -16,9 +16,12 @@ def __init__(self, base_path: Path, delim: str = "---"):
         self.path = ""
 
     def load(self, file: Path):
-        self.fm = []
```

Comment:
This shouldn't be running (at least through `academic`) until _after_ the file is made... I think there might be something downstream that's actually going wrong, not this.

Reply:
@christianbrodbeck Can you give some more context about how you ran this?

1. Where's your site located?
1. Where were you running this?

Reply:
The site is located at `~/Code/christianbrodbeck.net`, which is also the directory from which I ran the command. In case this matters, I'm running macOS 10.15.6, and installed go from the binaries on their website.

Reply:
Thanks for the context. :slightly_smiling_face: